nativescript-core/Application.md

```diff
@@ -0,0 +1,690 @@
+---
+title: Application
+---
+
+# Application
+
+The Application module provides abstraction over the platform-specific application implementations. The module lets you manage things such as handling lifecycle events of your application (cross-platform and/or native), sending Broadcasts on Android or adding a Notification observer on IOS.
+
+### Import
+
+/// flavor javascript
+
+```javascript
+import { Application } from '@nativescript/core'
+```
+
+///
+
+/// flavor typescript
+
+```typescript
+import { Application } from '@nativescript/core'
+```
+
+///
+
+## Android
+
+For android, you can access the main activity for the application, currently active activity, a method to register a broadcast receiver and other Android-specific properties and methods, as we will see below.
+
+## Android Properties
+
+### android
+
+The property gives you the Nativescript wrapper, the AndroidApplication object, around the native android native application instance.
+/// flavor javascript
+
+```javascript
+const androidApp = Application.android
+```
+
+///
+
+/// flavor typescript
+
+```typescript
+const androidApp: AndroidApplication = Application.android
+```
+
+///
+
+### nativeApp
+
+This a native application reference. Basically, it is the [android.app.Application](http://developer.android.com/reference/android/app/Application.html) instance keeping track of the global application state. From this object you can get methods such as `getFilesDir()`, `onLowMemory()`,etc.
+|||
+|----|-----|
+|Type:|`android.app.Application`|
+|||
+
+/// flavor javascript
+
+```javascript
+const nativeApp = androidApp.nativeApp
+```
+
+///
+/// flavor typescript
+
+```typescript
+const nativeApp: android.app.Application = androidApp.nativeApp
+```
+
+///
```

Comment:
Same as previous comment about grouping subsequent code blocks into tabs (a few more places like this, will not comment on each :) )

Reply:
Hi @rigor789. Thank you very much for your feedback. You just taught me new stuff :).  I will make the suggested changes.